internal/uvm/start.go

```diff
@@ -12,6 +12,8 @@ import (
 
 const _ERROR_CONNECTION_ABORTED syscall.Errno = 1236
 
+var _ = (OutputHandler)(parseLogrus)
```

Comment:
What is this needed for? Can you add a comment?

Reply:
This is standard go to make a compile time directive that `parseLogrus` implements `OutputHandler`. Once you see this enough you don't need a comment for it.

Reply:
Good to know!